osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceAttributes.cs

```diff
@@ -18,6 +18,9 @@ public class TaikoPerformanceAttributes : PerformanceAttributes
         [JsonProperty("effective_miss_count")]
         public double EffectiveMissCount { get; set; }
 
+        [JsonProperty("estimated_ur")]
+        public double? EstimatedUr { get; set; }
```

Comment:
Is it too late to ask to rename this to not use abbreviations? `EstimatedUnstableRate` if possible please.